frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -39,6 +39,7 @@ import { UPDATE_AOE } from "./addToQueue/AddToQueueSearch";
 export type TestResult = "POSITIVE" | "NEGATIVE" | "UNDETERMINED" | "UNKNOWN";
 
 const EARLIEST_TEST_DATE = new Date("01/01/2020 12:00:00 AM");
+const MAX_TEST_DATE = new Date();
```

Comment:
What if they have the website up for a few hours, this will be out-of-date

Reply:
Done, see https://github.com/CDCgov/prime-simplereport/pull/3156#discussion_r778377661